src/test/java/com/basho/riak/client/api/commands/CreateTableTest.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright 2013-2016 Basho Technologies Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.basho.riak.client.api.commands;
+
+import com.basho.riak.client.api.commands.timeseries.CreateTable;
+import com.basho.riak.client.core.operations.ts.CreateTableOperation;
+import com.basho.riak.client.core.query.timeseries.ColumnDescription;
+import com.basho.riak.client.core.query.timeseries.FullColumnDescription;
+import com.basho.riak.client.core.query.timeseries.TableDefinition;
+import com.basho.riak.protobuf.RiakTsPB;
+import org.junit.Test;
+import org.mockito.internal.util.reflection.Whitebox;
+
+import java.util.Arrays;
+import java.util.EnumSet;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+public class CreateTableTest extends OperationTestBase<CreateTableOperation>
+{
+    private static final TableDefinition tableDefinition = new TableDefinition("TestTable", Arrays.asList(
+            new FullColumnDescription("field1", ColumnDescription.ColumnType.TIMESTAMP,  false, 1),
+            new FullColumnDescription("field2", ColumnDescription.ColumnType.SINT64, false, null, 1),
```

Comment:
Do you think this case should result in an error when creating the TableDefinition? (Having 2 columns with the same key ordinal #).  The null ordinal # is confusing too...